app/retail/templates/home/index.html

```diff
@@ -306,12 +306,12 @@ <h3 class="mt-3 mb-2">{{ article.title }}</h3>
         </div>
       </div>
     </div>
+    {% include 'shared/bottom_notification.html' %}
+    {% include 'shared/analytics.html' %}
+    {% include 'shared/footer_scripts_lite.html' with slim=1 %}
+    {% include 'shared/footer.html' %}
+    {% include 'shared/messages.html' %}
```

Comment:
good maybe a good idea to move the other scripts also outside of body

Reply:
Actually, yes, but I have based myself on existing webpages:

https://github.com/gitcoinco/web/blob/6505bd5e4e67320deee0d28f64dee180b857cde5/app/dashboard/templates/bounty/increase.html#L162-L166
https://github.com/gitcoinco/web/blob/6505bd5e4e67320deee0d28f64dee180b857cde5/app/grants/templates/grants/cancel.html#L107-L111
https://github.com/gitcoinco/web/blob/6505bd5e4e67320deee0d28f64dee180b857cde5/app/grants/templates/grants/fund.html#L355-L359

Reply:
got it, yes good point we should fix that in some point (not for this PR)